dali/pipeline/operators/reader/loader/coco_loader.cc

```diff
@@ -301,10 +301,11 @@ void ParseAnnotationFilesHelper(std::vector<std::string> &annotations_filename,
     DALI_ENFORCE(f, "Could not open JSON annotations file");
     f.seekg(0, std::ios::end);
     size_t file_size = f.tellg();
-    std::unique_ptr<char, std::function<void(char*)>> buff(new char[file_size],
+    std::unique_ptr<char, std::function<void(char*)>> buff(new char[file_size + 1],
```

Comment:
`std::vector<char>` or even `std::string`?

Reply:
Done

Reply:
As said, I don't want to memset the allocated memory so no container can fit here.